.github/workflows/ci.yml

```diff
@@ -190,7 +190,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-22.04, ubuntu-24.04, ubuntu-24.04-arm, macos-13, windows-2022]
+        os: [ubuntu-latest, ubuntu-24.04-arm, macos-latest, windows-latest]
```

Comment:
Why remove ubuntu-22.04?

Reply:
We should just do the CI check on the latest. We also do the release binary build now as part of every PR, which tests against 22.04 (https://github.com/containerd/containerd/actions/runs/19080071262/job/54506009413?pr=12469).  We could probably ditch the arm build here since the release flow covers that as well.